src/react.js

```diff
@@ -78,7 +79,14 @@ module.exports = {
             allowGlobals: true
         }
     ],
+    'react/jsx-one-expression-per-line': [
+        'error',
+        {
+            allow: 'single-child'
```

Comment:
I'm assuming you're thinking this use case: 
```js
<App>Hello</App>
<App>{"Hello"}</App>
```

versus this one, which is meh. 

```js
<App><Hello /></App>
```
I don't know that I see this very often, though. Would Prettier take care of this anyway?

Reply:
Actually, all three of the ones you listed would be allowed with this rule configured this way.

https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/jsx-one-expression-per-line.md#allow

And yes, prettier would handle this, but this repo doesn't work with prettier. That's why [eslint-config-get-off-my-lawn-prettier](https://github.com/manovotny/eslint-config-get-off-my-lawn-prettier) exists (which would disable this rule), though I have ideas on how to make it all work in a single repo now. 😉